samples/index.md

```diff
@@ -1,18 +1,100 @@
+| [Flask](https://github.com/docker/awesome-compose/tree/master/flask){: target="_blank" rel="noopener" class="_"} | [Open in Docker Dev Environment](https://open.docker.com/dashboard/dev-envs?url=https://github.com/docker/awesome-compose/tree/master/flask) |
+| [Minecraft server](https://github.com/docker/awesome-compose/tree/master/minecraft){: target="_blank" rel="noopener" class="_"} | - |
+| [PHP](https://github.com/docker/awesome-compose/tree/master/apache-php){: target="_blank" rel="noopener" class="_"} | [Open in Docker Dev Environment](https://open.docker.com/dashboard/dev-envs?url=https://github.com/docker/awesome-compose/tree/master/apache-php) |
+| [Plex](https://github.com/docker/awesome-compose/tree/master/plex){: target="_blank" rel="noopener" class="_"} | - |
+| [Portainer](https://github.com/docker/awesome-compose/tree/master/portainer){: target="_blank" rel="noopener" class="_"} | - |
+| [Spark](https://github.com/docker/awesome-compose/tree/master/sparkjava){: target="_blank" rel="noopener" class="_"} | [Open in Docker Dev Environment](https://open.docker.com/dashboard/dev-envs?url=https://github.com/docker/awesome-compose/tree/master/sparkjava) |
+| [Traefik](https://github.com/docker/awesome-compose/tree/master/traefik-golang){: target="_blank" rel="noopener" class="_"} | [Open in Docker Dev Environment](https://open.docker.com/dashboard/dev-envs?url=https://github.com/docker/awesome-compose/tree/master/traefik-golang) |
+| [Wireguard](https://github.com/docker/awesome-compose/tree/master/wireguard){: target="_blank" rel="noopener" class="_"} | - |
+| [VueJS](https://github.com/docker/awesome-compose/tree/master/vuejs){: target="_blank" rel="noopener" class="_"} | [Open in Docker Dev Environment](https://open.docker.com/dashboard/dev-envs?url=https://github.com/docker/awesome-compose/tree/master/vuejs) |
+
+
+### Platform samples
+
+| Sample |
+| ------ |
+| [Gitea / PostgreSQL](https://github.com/docker/awesome-compose/tree/master/gitea-postgres){: target="_blank" rel="noopener" class="_"} |
+| [Nextcloud / PostgreSQL](https://github.com/docker/awesome-compose/tree/master/nextcloud-postgres){: target="_blank" rel="noopener" class="_"}
+| [Nextcloud / Redis / MariaDB](https://github.com/docker/awesome-compose/tree/master/nextcloud-redis-mariadb){: target="_blank" rel="noopener" class="_"} |
+| [Pi-hole / Cloudflared](https://github.com/docker/awesome-compose/tree/master/pihole-cloudflared-DoH){: target="_blank" rel="noopener" class="_"} |
+| [Prometheus / Grafana](https://github.com/docker/awesome-compose/tree/master/prometheus-grafana){: target="_blank" rel="noopener" class="_"} |
+| [Wordpress / MySQL](https://github.com/docker/awesome-compose/tree/master/wordpress-mysql){: target="_blank" rel="noopener" class="_"} |
+
+## Docker Samples on GitHub
```

Comment:
Wondering if it makes sense to move this to a new page to avoid a long scroll?

Reply:
Thanks, @usha-mandya. I tested splitting it into an "Awesome Compose" and a "Docker Samples" page with some internal users and the feedback was grouping by those concepts didn't make sense to them and why do they need to go to multiple pages to see, for example, all the React samples.

I will try splitting it into separate pages based on the service. Of course, samples with multiple services will be duplicated among the pages, so I'm not fully confident that it's a good solution.